components/gems/Gemfile.lock

```diff
@@ -394,6 +394,9 @@ GEM
     ffi (1.12.2)
     ffi (1.12.2-x64-mingw32)
     ffi (1.12.2-x86-mingw32)
+    ffi-compiler (1.0.1)
```

Comment:
@tas50 @lamont-granquist Is this going to bite me in the ass?

Reply:
🤷

Reply:
The new library which builds native extensions is https://github.com/cotag/http-parser which pulls in https://github.com/nodejs/http-parser/tree/77310eeb839c4251c07184a5db8885a572a08352 which is a C extension so... hopefully not?

![tenor-129399655](https://user-images.githubusercontent.com/2481463/80656752-c9feaf80-8a36-11ea-9d81-f8f0aea64d3b.gif)

Reply:
Turns out the answer is YES